menstruation/config.py

```diff
@@ -1,8 +1,106 @@
+import logging
+import os
+import sys
+from datetime import time, datetime
 from typing import Set, Optional, List
 import redis
 
 
-class MenstruationConfig:
+class MenstruationConfig(object):
+
+    def __init__(self) -> None:
+        args = self.get_environment_args()
+        self.__token: str = args['TOKEN']
+        self.__endpoint: str = args['ENDPOINT']
+        self.__redis_host: str = args['REDIS_HOST']
+        self.__moderators: list = args['MODERATORS']
+        self.__notification_time: time = args['NOTIFICATION_TIME']
+        self.__debug: bool = args['DEBUG']
+        self.user_db = UserDatabase(self.redis_host)
+        self.set_logging_level()
+
+    def get_token(self) -> str:
+        return self.__token
+
+    def get_endpoint(self) -> str:
+        return self.__endpoint
+
+    def get_redis_host(self) -> str:
+        return self.__redis_host
+
+    def get_moderators(self) -> list:
+        return self.__moderators
+
+    def set_moderators(self, moderators: List) -> None:
+        self.__moderators = moderators
+
+    def add_moderator(self, moderator: str) -> None:
+        self.__moderators.append(moderator)
+
+    def get_notification_time(self) -> time:
+        return self.__notification_time
+
+    def set_notification_time(self, new_time: time) -> None:
+        self.__notification_time = new_time
+
+    def get_debug(self) -> bool:
+        return self.__debug
+
+    def set_debug(self, debug: bool) -> None:
+        self.__debug = debug
+        self.set_logging_level()
```

Comment:
please dont write java in python. for properties, theres an `@property` decorator. or just use public attributes. these lines add nothing

Reply:
Well thanks, never headr form the decorator @property before.
In the meantime I have changed the object.